Gemfile

```diff
@@ -12,6 +12,7 @@ gem "autoforme", ">= 1.14"
 gem "aws-sdk-ec2", "~> 1.512"
 gem "aws-sdk-iam"
 gem "aws-sdk-s3"
+gem "aws-sdk-sts", "~> 1.12"
```

Comment:
This should be moved to it's own Gemfile group so users who are not using it do not have to load it. Probably best to use the same Gemfile group you are creating in #4714. To avoid conflicts, you may want to rebase this PR on top of that commit.

Reply:
Understood. A distinct group will be fine in that case.